packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -120,54 +94,12 @@ async function checkCapabilitiesFromStripe(
       ),
     ].filter(key => key in metadata);
     for (const key of capabilityKeys) {
-      const capabilities = metadata[key].split(',');
+      const capabilities = SubscriptionUtils.splitCapabilities(metadata[key]);
       for (const capability of capabilities) {
         capabilitiesToReveal.add(capability);
       }
     }
   }
-}
-
-// TODO: issue #3846 - remove this method
-async function fetchSubscribedProductsFromLocalDB(db, uid) {
-  const subscriptions = (await db.fetchAccountSubscriptions(uid)) || [];
-  const subscribedProducts = [
-    // All accounts get this psuedo-product
-    PRODUCT_REGISTERED,
-    ...subscriptions.map(({ productId }) => productId),
-  ];
-  // Accounts with at least one subscription get this product
-  if (subscribedProducts.length > 1) {
-    subscribedProducts.push(PRODUCT_SUBSCRIBED);
-  }
-  return subscribedProducts;
-}
-
-// TODO: issue #3913 - remove support for capabilities from server config
-function checkCapabilitiesFromServerConfig(
-  capabilitiesToReveal,
-  subscribedProducts,
-  client_id,
-  config,
-  auth
-) {
-  const {
-    subscriptions: { productCapabilities = {}, clientCapabilities = {} } = {},
-  } = config;
-
-  const subscribedCapabilities = subscribedProducts.reduce(
-    (capabilities, product) =>
-      capabilities.concat(productCapabilities[product] || []),
-    []
-  );
-
-  const clientVisibleCapabilities = clientCapabilities[client_id] || [];
 
-  subscribedCapabilities
-    .filter(
-      capability =>
-        auth.strategy === 'sessionToken' ||
```

Comment:
Removing this. The check for `sessionToken` auth type here was intended for use by content-server. But, we didn't end up using it.

Instead, content-server frontend queries auth-server for at least one subscription (not capabilities) to decide whether to show the management button.